src/sentry/relay/config/metric_extraction.py

```diff
@@ -841,6 +841,9 @@ def _generate_span_attribute_specs(project: Project) -> list[HashedMetricSpec]:
         except ValueError:
             logger.exception("Invalid span attribute metric spec", extra=rule.to_dict())
 
+    max_specs = options.get("metric_extraction.max_span_attribute_specs")
+    (specs, _) = _trim_if_above_limit(specs, max_specs, project, "span_attributes")
```

Comment:
Not for this PR, but we should log that somewhere so we notice if the limit was reached.

Reply:
`_trim_if_above_limit` already does that for us with a [capture_exception](https://github.com/getsentry/sentry/blob/ogi/feat/span-attribute-spec-limit/src/sentry/relay/config/metric_extraction.py#L376)